docs/dev/system_keyspace.md

```diff
@@ -364,7 +364,7 @@ Schema:
 ~~~
 CREATE TABLE system.sstables (
     location text,
-    generation bigint,
+    generation timeuuid,
```

Comment:
nit: This change doesn't belong to `docs/dev/system_keyspace: add a new section for non-volatile tables` PR title

Reply:
agreed. extracted into #13994